RELEASE/scripts/autoscend/auto_familiar.ash

```diff
@@ -11,6 +11,63 @@ boolean is100FamRun()
 	return true;
 }
 
+boolean isHinderingFamiliar100Run()
+{
+	//indicates that we are in a 100% familiar run with a familiar that hinders you. and thus should not be buffed
+	if(!is100FamRun())
+	{
+		return false;
+	}
+	familiar hundred_fam = to_familiar(get_property("auto_100familiar"));
+	
+	//these familiars are only harmful
+	if($familiars[black cat, O.A.F.] contains hundred_fam)
+	{
+		return true;
+	}
+	
+	//these familiars sometime attack the enemy
+	if($familiars[Fuzzy Dice, Stab Bat, Killer Bee, Scary Death Orb, RoboGoose] contains hundred_fam)
+	{
+		return true;
+	}
+	
+	return false;
```

Comment:
Maybe also treat pet rock type familiars as hindering since buffing them effectively does nothing

Reply:
Oh yes. thanks for the reminder. I had planned on doing so, then did some other stuff and forgot about it

Reply:
added pet rock type familiars as sugggested